ansible/roles/prosody/tasks/install.yml

```diff
@@ -64,6 +58,44 @@
     - inspect
     - uuid
 
+
+- name: Install lua 5.4
+  ansible.builtin.apt:
+    name: ["lua5.4", "liblua5.4-dev"]
+    state: present
+
+- name: Set default lua 5.4 and luarocks to use 5.4
```

Comment:
Does this also need a `when: prosody_use_lua_54` like the one in configure.yml?

Reply:
Looks like no :)

Reply:
Nope as this is install time, we want during configure to have the option to chose, so on install we install both.